dynamic-property-jackson/src/test/java/ru/fix/dynamic/property/jackson/ComposableDynamicPropertyMarshallerTest.java

```diff
@@ -11,9 +12,9 @@
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
-class JacksonDynamicPropertyMarshallerTest {
+class ComposableDynamicPropertyMarshallerTest {
 
-    private final JacksonDynamicPropertyMarshaller marshaller = new JacksonDynamicPropertyMarshaller();
+    private final DynamicPropertyMarshaller marshaller = MarshallerBuilder.newBuilder().build();
```

Comment:
Maybe create some trivial helper class and wrap `MarshallerBuilder.newBuilder().build()` as static method. For example

```java
class DefaultJacksonMarshaller {
      DynamicPropertyMarshaller get() {
                return MarshallerBuilder.newBuilder().build()
       }
}
```

What do you think about it? In most cases we use default preconfigured marshaller